src/jobs/summary-notification.ts

```diff
@@ -226,3 +236,29 @@ export async function printSummary(cliConfig: IConfig) {
     log.error(`Summary notification: ${error.message}`)
   }
 }
+
+function createTweetMessage({
+  number_of_probes,
+  average_response_time,
+  number_of_incidents,
+  number_of_recoveries,
+}: {
+  number_of_probes: number
+  average_response_time: number
+  number_of_incidents: number
+  number_of_recoveries: number
+}): string {
+  const message = `I'm using Monika by @hyperjump_tech to monitor ${number_of_probes} probes! In the last 24 hours, 
+
+⏱ the average response time is ${average_response_time} ms
+⚠️ there were ${number_of_incidents} incidents
+✅ and ${number_of_recoveries} recoveries!
+
+Give it a try!
+
+opensource
```

Comment:
Is this supposed to be a hashtag or just plain text?

Reply:
fixed by adding opensource to hashtags